osm-fudge/__main__.py

```diff
@@ -0,0 +1,27 @@
+import bk_tree
+import sys
+
+import levenshtein
+import jaccard
+
+distance_metrics = {
+	'levenshtein': levenshtein.find_levenshtein_distance,
+	'jaccard': jaccard.find_jaccard_similarity
+}
+
+with open(sys.argv[1], 'r') as file:
+	initial_word = file.readline()
+	metrics = sys.argv[2]
+	if metrics not in distance_metrics.keys():
+		sys.exit("Allowed metrics are {metrics}".format(metrics=list(distance_metrics.keys())))
+	options = {'n' : 2}
+	tree_obj = bk_tree.BKTree(initial_word, distance_metrics[metrics], options)
+
+	for line in file: 
+		if len(line) > 1:
+			tree_obj.insert(line.strip(), tree_obj.tree)
```

Comment:
Not sure why you indent sometimes with tabs, sometimes with spaces :)

Usually you use four spaces for everything Python related.

Reply:
Yes, my sublime editor keeps changing between tabs and spaces. Can we use linter for python?

Reply:
Hm, I used flake8 before as a linter; not sure if it's also catching spaces vs tabs :thinking: 

Maybe ticket out integrating flake8 and enabling Travis for this repository.

And then let Travis run flake8 in the continuous builds?

Reply:
Yes, sure gonna make a PR for it. 👍